alfred-telemetry-platform/src/main/resources/alfresco/module/alfred-telemetry-platform/alfresco-global.properties

```diff
@@ -54,3 +54,8 @@ alfred.telemetry.binder.legacy-memory.enabled=false
 alfred.telemetry.binder.legacy-gc.enabled=false
 alfred.telemetry.binder.legacy-cache.enabled=false
 alfred.telemetry.binder.legacy-jdbc.enabled=false
+
+## Support for Solr metrics
+eu.xenit.alfred.telemetry.binder.solr.sharding.cronexpression=0/10 * * * * ?
```

Comment:
Prefixed with `eu.xenit.`. Might indeed be better, however we should be consistent?

Reply:
right